manager/notifier.go

```diff
@@ -87,6 +87,7 @@ var (
 	smtpSender             = flag.String("notification.smtp.sender", "alertmanager@example.org", "Sender email address to use in email notifications.")
 	hipchatURL             = flag.String("notification.hipchat.url", "https://api.hipchat.com/v2", "HipChat API V2 URL.")
 	flowdockURL            = flag.String("notification.flowdock.url", "https://api.flowdock.com/v1/messages/team_inbox", "Flowdock API V1 URL.")
+	slackConnectTimeout    = flag.Int("notification.slack.timeout", 5, "http timeout to talk to slack (in seconds).")
```

Comment:
s/http/HTTP/

If 5 was not enough in your case, can we set the default to at least 10 or something?